### ISSUE TITLE
"interface" in file context definition

### ISSUE THREAD

**Issue by vmojzis** [SELinuxProject/selinux#266]
String "interface" at the end of file context definition breaks the policy module ("interface(anything)" leads to the same).
```
Reproducer:
# cat test.te << "policy_module(test, 1.5.0)"
# cat test.fc << "/interface              gen_context(system_u:object_r:tmp_t,s0)"
# make -f /usr/share/selinux/devel/Makefile test.pp
# semodule -i test.pp
libsemanage.semanage_pipe_data: Child process /usr/libexec/selinux/hll/pp failed with code: 255. (No such file or directory).
test: Invalid mode in file context line: / 	 	define(`',` dnl
libsemanage.semanage_direct_commit: Failed to compile hll files into cil files.
 (No such file or directory).
```

It seems that the string "interface(anything)" is recognised as interface definition and hence gets replaced by "define(`anything',`"

**Comment by WOnder93** [comment]
This is more likely a bug in the policy build system (Makefile + scripts), rather than the userspace programs.

**Comment by cgzones** [comment]
Probably you need to insert a protected whitespace, like:

https://github.com/SELinuxProject/refpolicy/blob/493f56b59de002c93d0b77a9e7b2df8f3b752ed8/policy/modules/kernel/files.fc#L200-L201

**Comment by vmojzis** [comment]
> Probably you need to insert a protected empty string, like:
> 
> https://github.com/SELinuxProject/refpolicy/blob/493f56b59de002c93d0b77a9e7b2df8f3b752ed8/policy/modules/kernel/files.fc#L200-L201

Yes, thank you, in my case I need to disrupt the word "interface" in any way, so "interfac`'e", or "interfac(e)" will work.
